### ISSUE TITLE
Deprecation warning from Formik

### ISSUE THREAD

**Issue by ziaulrehman40** [SatelCreative/formik-polaris#51]
Hi there, thanks for the nice wrapper, can you please also fix this warning. Doesn't seem like a lot of change.

Thanks.

```
<Field render> has been deprecated and will be removed in future versions of Formik. Please use a child callback function instead. To get rid of this warning, replace <Field name="company_name" render={({field, form}) => ...} /> with <Field name="company_name">{({field, form, meta}) => ...}</Field>
```

**Comment by qw-in** [comment]
@ziaulrehman40 I have released `0.4.0` which should fix this warning. I am also planning a major release to better support `formik@2` soon

**Comment by ziaulrehman40** [comment]
That's good to know :+1:
